minimint/src/config.rs

```diff
@@ -89,9 +89,9 @@ impl GenerateConfig for ServerConfig {
 
         let fee_consensus = FeeConsensus {
             fee_coin_spend_abs: minimint_api::Amount::ZERO,
-            fee_peg_in_abs: minimint_api::Amount::from_sat(500),
+            fee_peg_in_abs: minimint_api::Amount::from_sat(1000),
```

Comment:
I guess we finally need module-governed fees to adapt them to mempool conditions.